src/components/StatsBoxGrid.js

```diff
@@ -182,11 +182,11 @@ const GridItem = ({ metric, dir }) => {
         margin={{ left: -5, right: -5 }}
       >
         <defs>
-          <linearGradient id="colorUv" x1="0" y1="0" x2="0" y2="1">
+          <linearGradient id="colorUv-${chartType}" x1="0" y1="0" x2="0" y2="1">
             <stop offset="5%" stopColor="#8884d8" stopOpacity={1} />
             <stop offset="95%" stopColor="#8884d8" stopOpacity={0} />
           </linearGradient>
-          <linearGradient id="colorPv" x1="0" y1="0" x2="0" y2="1">
+          <linearGradient id="colorPv-${chartType}" x1="0" y1="0" x2="0" y2="1">
```

Comment:
`chartType` var doesn't exist. I used it just as an example in the issue description.

Perhaps we could use the `title` here that is being destructured in line 150. For safety reasons, you could do: `_.kebabCase(title)` to avoid having spaces.

What do you think?

Reply:
We already have `lodash` installed. You can just do `import { kebabCase } from "lodash"` 👍🏼